shared/folders/dumb.js

```diff
@@ -335,6 +337,16 @@ export const files: DumbComponentMap<Files> = {
     'You can unlock - Private': {
       ...commonFiles(true),
       ...commonUnlock
+    },
+    'Feature Disabled - Private': {
+      ...commonFiles(true),
+      recentFilesSection: undefined,
```

Comment:
:sheep: Nit: perhaps use `null` instead of `undefined` for empty values since undefined can be ambiguous with nonexistent properties?